api/src/main/java/com/epam/pipeline/entity/metadata/CategoricalAttributeValue.java

```diff
@@ -21,29 +21,35 @@
 import lombok.EqualsAndHashCode;
 import lombok.NoArgsConstructor;
 import java.util.Collections;
+import java.util.Date;
 import java.util.List;
 
 @Data
 @NoArgsConstructor
 @AllArgsConstructor
-@EqualsAndHashCode(exclude = {"id"})
+@EqualsAndHashCode(of = {"key", "value", "autofill", "links"})
 public class CategoricalAttributeValue {
 
     private Long id;
     private String key;
     private String value;
     private Boolean autofill;
+    private Long attributeId;
+    private String owner;
```

Comment:
I'd suggest to keep `createdDate` and `owner` in `CategoricalAttribute` class

Reply:
Done, changed mapping process in DAO to remove those fields